README.md

```diff
@@ -8,39 +10,25 @@
 
 *The graph database for delightful React state management*
 
-## Installation
```

Comment:
I think this should stay at the top. I often want a reminder of the install instructions even for packages I'm already using in other projects. And I get annoyed by readmes that burry these and make me read

Reply:
makes sense! I tried moving it under 'features' right now based on the format I've been seeing in popular projects I like of pitching the library before giving the install instructions. Is that high enough?

Reply:
ok I ended up moving it under roadmap as well but I'm still split on it. I do think we need to pitch what we're doing more better and faster and I think we should prioritize that given the goal right now is to get new users. Does that make sense or still bothered?